src/test/java/bio/terra/app/controller/SnapshotValidationTest.java

```diff
@@ -170,9 +170,9 @@ public void testSnapshotNameInvalid() throws Exception {
         errorModel = expectBadSnapshotCreateRequest(snapshotByAssetRequest);
         checkValidationErrorModel(errorModel, new String[]{"Size", "Pattern"});
 
-        // Make a 64 character string, it should be considered too long by the validation.
-        // Note: a 63 character string, we are okay with
-        String tooLong = StringUtils.repeat("a", 64);
+        // Make a 512 character string, it should be considered too long by the validation.
+        // Note: a 511 character string, we are okay with
+        String tooLong = StringUtils.repeat("a", 512);
```

Comment:
There's probably a way to get this info from the validation annotation on the java class, but I don't know how to do it.

Reply:
So I looked into this a bit and there's definitely a way to do it, but I haven't quite figured it out either. I'll make a follow-up ticket since I agree it would be a nice improvement 🙂